### PR TITLE
feat(app): replace lookup with search with google in electron-context-menu

### LINKED ISSUE #11220: update electron-context-menu
update electron-context-menu to use `Google Search` instead of `lookup`

### PR DESCRIPTION
**Pull request by koji**
# Overview
This PR will replace lookup with search google in electron-context-menu.
fix #11220 and fix [RAUT-66](https://opentrons.atlassian.net/jira/software/c/projects/RAUT/boards/75?modal=detail&selectedIssue=RAUT-66)

# Changelog
- update electron-context-menu
- replace lookup method with search google method

# Review requests
`This PR will need to install electron-context-menu (^3.3.0) to run app.`
`make teardown-js then make setup-js`
- [ ] when turn on devtools, the menu shows copy, search with google, and inspect element
- [ ] when turn off devtools, the menu shows copy, search with google no `inspect element`

maybe need to use build app
https://s3.amazonaws.com/opentrons-app/builds/Opentrons-v6.0.0-mac-b20785-app_feat-replace-electron-context-menu.dmg

# Risk assessment
low

### CODE REVIEW

**Review by ecormany — CHANGES_REQUESTED** [review]
This appears to be incompatible with the React Select update merged in #11104. I get this error when `make setup-js` was most recently done from `edge`:

```
[shell] 10:08:27.956 [main] error: Uncaught:  {
[shell]   error: TypeError: actions.searchWithGoogle is not a function
[shell]       at Object.menu (webpack-internal:///./src/main.ts:74:49)
[shell]       at Object.<anonymous> (/Users/edwardcormany/opentrons/app-shell/node_modules/electron-context-menu/index.js:180:27)
[shell]       at Object.emit (events.js:315:20)
[shell] }
```

The error happens every time I right-click (and no contextual menu appears).

If I do teardown and setup on this branch, everything works as expected.

**Comment by jerader** [comment]
> This appears to be incompatible with the React Select update merged in #11104. I get this error when `make setup-js` was most recently done from `edge`:
> 
> ```
> [shell] 10:08:27.956 [main] error: Uncaught:  {
> [shell]   error: TypeError: actions.searchWithGoogle is not a function
> [shell]       at Object.menu (webpack-internal:///./src/main.ts:74:49)
> [shell]       at Object.<anonymous> (/Users/edwardcormany/opentrons/app-shell/node_modules/electron-context-menu/index.js:180:27)
> [shell]       at Object.emit (events.js:315:20)
> [shell] }
> ```
> 
> The error happens every time I right-click (and no contextual menu appears).
> 
> If I do teardown and setup on this branch, everything works as expected.

I think @koji might need to rebase edge and then this error might fix itself @ecormany

**Review by vabruzzo — APPROVED** [review]

File: app-shell/src/main.ts [hunk 1]
```diff
@@ -65,14 +65,12 @@ function startUp(): void {
...
-  // TODO kj: remove {} once https://github.com/sindresorhus/electron-context-menu/issues/153 is solved
-  // After upgrading electron-context-menu to after 0.16.0 remove lookUpSelection() and add searchWithGoogle()
   contextMenu({
-    menu: actions => [
-      actions.copy({}),
-      actions.lookUpSelection({}),
-      actions.inspect(),
-    ],
+    menu: actions => {
+      return config.devtools
```
Comment (koji) [inline app-shell/src/main.ts:70]: enabled devtools: copy, search with google, and inspect element
disabled devtools: copy and search with google

**Review by ecormany — APPROVED** [review]
Forgot I had to do `make teardown-js setup-js`…but it's working now! 🚢

**Pull request merged**